app/src/main/java/org/asteroidos/sync/PermissionsActivity.java

```diff
@@ -34,7 +35,11 @@ protected void onCreate(@Nullable Bundle savedInstanceState) {
         PackageManager pm = getPackageManager();
         boolean hasBLE = pm.hasSystemFeature(PackageManager.FEATURE_BLUETOOTH_LE);
 
-        if (hasBLE) {
+        ActivityManager am = (ActivityManager) getApplicationContext().getSystemService(ACTIVITY_SERVICE);
+        System.out.println("is low ram: " + am.isLowRamDevice());
+        boolean isLowRamDevice = am.isLowRamDevice();
```

Comment:
maybe you should use this variable in the previous line too ?

Reply:
oh my debug function is still in there. Good catch.